examples/blog-starter/static/feed.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
What's this file for ? looks like the output of `rss-feed.js`, but the feed is automatically generated so I don't think we need it here for the example

Reply:
Yes. This is a leftover. I'll remove it.